README.adoc

```diff
@@ -160,39 +178,49 @@ java -jar target/time-source-rabbit-3.0.0-SNAPSHOT.jar \
      --task.launch.request.task-name-expression="'task-'+headers['task-id']"
 ```
 
-The transformed message looks like this:
+Now, the transformed message would look like:
 
 ```
 headers:
-task-id:	34
-content_type:	application/json
+task-id:   34
+content_type:  application/json
 Payload
 49 bytes
 Encoding: string
 {"args":[],"deploymentProps":{},"name":"task-34"}
 ```
 
-Let's upack this function definition:
+Let us unpack what is happening behind the scenes. We will start with the following function definition.
 
 `timeSupplier|spelFunction|headerEnricherFunction|taskLaunchRequestFunction`
 
-This creates a composed Supplier beginning with the default `timeSupplier` which is the foundation for `time-source`.
-This produces a String like `06/18/20 16:01:38`.  We transform this using the SpEL expression `payload.length()`.
-The `spelFunction` applies to a `Message` from which we can extract and transform the `payload` or `headers`, in accordance with common Spring Integration conventions.
+- Here, the function definition creates a composite Supplier beginning with the default `timeSupplier` Java function included
```

Comment:
Probably the `Supplier` word has to be as a code...

Reply:
Is "Supplier" the name of a code element (class, method, interface, or some other code structure)? If so, it should be marked as code. If not, it should be uncapitalized ("supplier") to avoid this confusion.